apps/hash-ai-worker-ts/src/activities/flow-activities/write-google-sheet-action.ts

```diff
@@ -282,11 +282,15 @@ export const writeGoogleSheetAction: FlowActionActivity<{
             },
           },
         },
-        ...existingSheets.map((sheet) => ({
-          deleteSheet: {
-            sheetId: sheet.properties?.sheetId,
-          },
-        })),
+        ...existingSheets
+          .filter(
+            (sheet) => sheet.properties?.sheetId !== placeholderFirstSheetId,
+          )
+          .map((sheet) => ({
+            deleteSheet: {
+              sheetId: sheet.properties?.sheetId,
+            },
+          })),
```

Comment:
I think so? At least I got the same thing and added this and haven't seen it since, although I'm not sure how the sheets are ending up in a position where they have the placeholder in them